pkg/util/priority/priority_test.go

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package priority
+
+import (
+	"context"
```

Comment:
format the import as other files.

Reply:
Could we run `make ci-lint` in the CI? If you give me a pointer to where the jobs are defined, I'm happy to add the job to the pipeline

Reply:
We have added `ci-lint` in `verify`. But I think it doesn't identify this importing format.
https://github.com/kubernetes-sigs/kueue/blob/99683618876a004f3332fb99cae1e3a9bf6f2124/Makefile#L125

Reply:
I see, I think we can detect it by updating gonalgci-lint configuration so let me work on in another PR